test/tests/functional/pbs_qstat.py

```diff
@@ -102,3 +102,42 @@ def test_qstat_qselect(self):
                               as_script=True)
         if ret['rc'] != 0:
             self.assertFalse(ret['err'][0], ret_msg)
+
+    def test_qstat_n_ip(self):
+        self.server.manager(MGR_CMD_DELETE, NODE, None, '')
+        ipaddr = socket.gethostbyname(self.mom.hostname)
+        attr_A = {'Mom': self.mom.hostname}
+        self.server.manager(MGR_CMD_CREATE, NODE, id=ipaddr, attrib=attr_A)
+        self.server.expect(NODE, {'state=free': 1})
+        j = Job(TEST_USER)
+        jid = self.server.submit(j)
+        self.server.expect(JOB, {'job_state': 'R'}, id=jid)
+        qstat_cmd = \
+            os.path.join(self.server.pbs_conf['PBS_EXEC'], 'bin', 'qstat')
+        qstat_cmd_n = [qstat_cmd, '-n', str(jid)]
+        ret = self.du.run_cmd(self.server.hostname, cmd=qstat_cmd_n)
+        self.assertEqual(ret['rc'], 0,
+                         'Qstat returned with non-zero exit status')
+        qstat_out = '\n'.join(ret['out'])
+        self.assertIn(ipaddr, qstat_out,
+                      "Incorrect node name in qstat -n when "
+                      "node created using IP address")
+
+    def test_qstat_n_fqdn(self):
+        self.server.manager(MGR_CMD_DELETE, NODE, None, '')
+        self.server.manager(MGR_CMD_CREATE, NODE, id=self.mom.hostname)
+        self.server.expect(NODE, {'state=free': 1})
+        j = Job(TEST_USER)
+        jid = self.server.submit(j)
+        self.server.expect(JOB, {'job_state': 'R'}, id=jid)
+        qstat_cmd = \
+            os.path.join(self.server.pbs_conf['PBS_EXEC'], 'bin', 'qstat')
```

Comment:
Please avoid the use of backslash for line continuation where possible in Python. Align the last two arguments one space to the right of the open parenthesis on the next line and pycodestyle will be happy.